src/libgap-api.c

```diff
@@ -258,3 +258,16 @@ char * GAP_CSTR_STRING(Obj string)
         return 0;
     return CSTR_STRING(string);
 }
+
+UChar GAP_CharValue(Obj obj)
+{
+    if (TNUM_OBJ(obj) != T_CHAR) {
+        ErrorQuit("GAP_CharValue: argument is not a char", 0, 0);
+    }
+    return CHAR_VALUE(obj);
+}
+
+Obj GAP_GetChar(UChar obj)
```

Comment:
Maybe GetCharObj, just so this isn't confused with the C function getchar (which reads a char from the terminal).

Reply:
Maybe `GAP_ValueOfChar` and `GAP_CharWithValue` ?